Challenges/madlibs.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+
+# An title is needed for the project
+
+puts "--------------------------"
+puts "|   Power of the force    |"
+puts "--------------------------"
+
+puts "Give me an Adjective?"
+puts ">>>"
+adj1 = gets.chomp
+puts "Give me a Plural Noun?"
+puts ">>>"
+p_noun1 = gets.chomp
+puts " Give me a Noun?"
+puts ">>>"
+noun1 = gets.chomp
+puts "Give me an Adjective?"
+puts ">>>"
+adj2 = gets.chomp
+puts "Give me a part of your body?"
+puts ">>>"
+part1 = gets.chomp
+puts "Give me a adjective?"
+puts ">>>"
+adj3 = gets.chomp
+puts "Give me a Plural Noun?"
+puts ">>>"
+p_noun2 = gets.chomp
+puts "Give me a part of the body?"
+puts ">>>"
+part2 = gets.chomp
+puts "Give me an Adjective?"
+puts ">>>"
+adj4 = gets.chomp
+puts "Give me an Adverb?"
+puts ">>>"
+adv = gets.chomp
```

Comment:
Looks like there's a lot of repeated code here. Can we extract some of this into a method? Defining a method would allow us to write something like:
```ruby
adj1 = get_user_input("adjective")
```
where `get_user_input` is a method that asks the user for the type of word you pass in and returns whatever the user enters.